proxy/http2/Http2ConnectionState.h

```diff
@@ -220,7 +220,7 @@ class Http2ConnectionState : public Continuation
   void send_data_frames(Http2Stream *stream);
   Http2SendADataFrameResult send_a_data_frame(Http2Stream *stream, size_t &payload_length);
   void send_headers_frame(Http2Stream *stream);
-  void send_push_promise_frame(Http2Stream *stream, URL &url);
+  void send_push_promise_frame(Http2Stream *stream, URL &url, MIMEField *accept_encoding);
```

Comment:
It would be great if this is `const MIMEField *accept_encoding` for const correctness. You don't change this MIMEField, right?